library/components/TTooltip.vue

```diff
@@ -38,17 +37,15 @@ export default {
 			type: String,
 			default: "top"
 		},
+		open: { // To control state from a component.
```

Comment:
I'd like to use `isOpen` for boolean states. Summarised how we can named boolean props:

Use prefix is, can, has:

-  is: for visual/behavior variations. e.g. isVisible, isEnable, isActive
- can: fore behavior variations or conditional visual variations. e.g. canToggle, canExpand, canHaveCancelButton
- has: for toggling UI elements. e.g. hasCancelButton, hasHeader

Reply:
Yes, probably best to use that prefix because it is just like our other components, @sareg0 agree?

Reply:
sounds good